package.json

```diff
@@ -33,8 +33,8 @@
     "mocha": ""
   },
   "dependencies": {
-    "agentkeepalive": "^4.1.3",
-    "arsenal": "scality/Arsenal#918a1d7",
+    "agentkeepalive": "^4.1.4",
+    "arsenal": "scality/Arsenal#918a1d7c899a7ecddcd1d6e7bbdd49b5becc5bd0",
```

Comment:
There should be a tag for arsenal instead of using the git hash, if not we can make one

Reply:
I guess there is no tag associated to this specific commit, `git show-ref --tags | grep 918a1d7c899a7ecddcd1d6e7bbdd49b5becc5bd0` returns nothing. Would you agree then a tag should be created?

Reply:
Hmm, there is probably a later tag to use that should be fine. Depends on what base branch is needed, could be 7x or 8x I don't know.